addons/website_sale/res_config.py

```diff
@@ -0,0 +1,8 @@
+from openerp import models, fields
```

Comment:
Do not forget to put a first comment line with your file encoding: 
```python
# -*- encoding: utf-8 -*-
```
It will avoid stupid crashes if we have to use non-ASCII chars